core/src/main/scala/com/yahoo/maha/core/query/druid/DruidQueryGenerator.scala

```diff
@@ -729,10 +728,13 @@ class DruidQueryGenerator(queryOptimizer: DruidQueryOptimizer
         }
       }
 
-      queryContext.requestModel.orFilterMeta.foreach{
-        meta =>
-          //val dimMeta = meta.orFilter.filters.filter()
-          if(!(meta.filterType == MetaType.FactType)) outerQueryDimFilterList += FilterDruid.renderOrDimFilters(meta.orFilter.filters, aliases.toMap, cols, Option.empty, true)
+      val orFilterTypes = queryContext.requestModel.orFilterMeta.map {
```

Comment:
why do you need this? 
can't you just do 
`queryContext.requestModel.orFilterMeta.filter(f => f.filterType == MetaType.DimType).foreach {`